include/drivers/mmc.h

```diff
@@ -36,6 +36,20 @@
 #define OCR_VDD_MIN_2V0			GENMASK(14, 8)
 #define OCR_VDD_MIN_1V7			BIT(7)
 
+#define MMC_RSP_48			(1 << 0)
```

Comment:
The defines here from line 39 to 43 introduces some MISRA defects like 
`drivers/mmc/mmc.c:336:[MISRA C-2012 Rule 10.1 (required)]<275747> The operand "1" to the operator "<<" does not have an essentially unsigned type.`

Suggest to use the `BIT()` macro for these defines.